common/src/test/java/org/apache/uniffle/common/config/ConfigOptionTest.java

```diff
@@ -131,6 +131,20 @@ public void testListTypes() {
     } catch (IllegalArgumentException illegalArgumentException) {
       fail();
     }
+
+    // test the empty list
+    final ConfigOption<List<String>> emptyListStringOption = ConfigOptions
+            .key("rss.key5")
+            .stringType()
+            .asList()
+            .noDefaultValue()
+            .withDescription("List config key5");
```

Comment:
If we don't set this option and  call method conf.get(emptyListStringOption) directly, what will the result be?

Reply:
It will return null.

Reply:
Is the same behavior as Flink?

Reply:
Yes. I have checked.